subworkflows/yaml-schema.json

```diff
@@ -0,0 +1,93 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "Meta yaml",
+    "description": "Validate the meta yaml file for an nf-core subworkflow",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "Name of the subworkflow"
+        },
+        "description": {
+            "type": "string",
+            "description": "Description of the subworkflow"
+        },
+        "authors": {
+            "type": "array",
+            "description": "Authors of the subworkflow",
+            "items": {
+                "type": "string"
+            }
+        },
+        "modules": {
+            "type": "array",
+            "description": "Modules used in the subworkflow",
+            "items": {
+                "type": "string"
+            }
+        },
+        "keywords": {
+            "type": "array",
+            "description": "Keywords for the module",
+            "items": {
+                "type": "string"
+            },
+            "minItems": 3
+        },
+        "input": {
+            "type": "array",
+            "description": "Input channels for the subworkflow",
+            "items": {
+                "type": "object",
+                "patternProperties": {
+                    ".*": {
+                        "type": "object",
+                        "properties": {
+                            "type": {
+                                "type": "string",
+                                "description": "Type of the input channel"
+                            },
+                            "description": {
+                                "type": "string",
+                                "description": "Description of the input channel"
+                            },
+                            "pattern": {
+                                "type": "string",
+                                "description": "Pattern of the input channel, given in Java glob syntax"
+                            }
```

Comment:
We have also a "default" field in some of them, do we want to check this one too?

Reply:
Good to know, I'll add it back then.